azure-pipelines.yml

```diff
@@ -94,6 +94,63 @@ stages:
 
   #######################################
 
+  - job: RPIBuilds
+    pool:
+      name: Default
+      demands:
+        - agent.name -equals iio-osc_rpi
+    workspace:
+      clean: all
+    steps:
+    - checkout: self
+      fetchDepth: 1
+      clean: true
+      persistCredentials: true
+    - task: DownloadPipelineArtifact@2
+      inputs:
+        source: 'specific'
+        project: '$(System.TeamProjectId)'
+        pipeline: $(libiioPipelineId)
+        artifact: 'Ubuntu-arm32v7'
```

Comment:
Why are you using artifacts from Ubuntu inside RPI build (line #114, #124, #134 etc)?
Even if that would work, it is not ok.